notebooks/Training a MONAI Segmentation Bundle.ipynb

```diff
@@ -435,9 +503,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Job Log Download\n",
+    "### **1. Job Log Download**\n",
```

Comment:
If possible, I would prefer not to add numbers and `**`  because formats are difficult to track across versions and notebooks.

Reply:
I am not sure about this. If this is a serious issue, I will refactor all these kind of  `** + number` pattern at the end.